sdks/router-sdk/src/entities/mixedRoute/route.ts

```diff
@@ -54,17 +55,22 @@ export class MixedRouteSDK<TInput extends Currency, TOutput extends Currency> {
       const prevPool = pools[i - 1]
       const pool = pools[i]
       const inputToken = tokenPath[i]
+      const isWrappedNativeEquality = ((pool.token0.equals(inputToken.wrapped) || pool.token1.equals(inputToken.wrapped)) && inputToken.isNative)
+      || ((pool.token0.wrapped.equals(inputToken) && pool.token0.isNative) || (pool.token1.wrapped.equals(inputToken) && pool.token1.isNative))
```

Comment:
`pool.token1.isNative` this is never true because the native token is address(0), and token0 < token1, so it is always token0